plugins/modules/event.py

```diff
@@ -97,23 +97,6 @@
         description:
           - The total state change percentage for the check's history.
         type: int
-  metric_attributes:
```

Comment:
Is this attribute deprecated inside SensuGo? If it is, would like to know from which version. For compatibility reasons I I would leave it in, since it is not mandatory at all. In case that It must be removed, I will remove it also from argument specs.

Reply:
@juremedvesek I got a little overzealous in removing that section. It hasn't been deprecated, but the link has moved to https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metrics-attribute.

Reply:
Should be good to go now.